install/pre-installation.md

```diff
@@ -522,38 +522,58 @@ Run the following steps before starting any of the system configuration procedur
 
 1. (`pit#`) Download the SHCD to the `prep` directory.
 
-   This will need to be retrieved from the administrators Cray deliverable.
+    This will need to be retrieved from the administrators Cray deliverable.
 
 1. Validate the SHCD.
 
-   See [Validate SHCD](../operations/network/management_network/validate_shcd.md) and then return to this page.
+    See [Validate SHCD](../operations/network/management_network/validate_shcd.md) and then return to this page.
 
 ### 3.2 Generate topology files
 
-1. (`pit#`) Generate `hmn_connections.json`.
+The following steps use the new, automated method for generating files. The previous step for
+[validate SHCD](#31-validate-shcd) will have generated "paddle" files, these are necessary for generating
+the rest of the seed files.
 
-   ```bash
-   csi config shcd "$SYSTEM_NAME-hmn-paddle.json" -H
-   ```
+> ***NOTE*** The paddle files are temporarily not used due to bugs in the seed file generation software.
+> Until these bugs are resolved, the seed files must be manually generated.
+
+If seed files from a prior installation of the same major-minor version of CSM exist, these can be used and
+this step may be skipped.
 
-1. (`pit#`) Create `application_node_config.yaml`, `ncn_metadata.csv`, and `switch_metadata.csv`.
+1. (`pit#`) Create each seed file unless they already exist from a previous installation:
 
-    ```bash
-    csi config shcd "${SYSTEM_NAME}-full-paddle.json" -ANS
-    ```
+   - For new installations of CSM that have no prior seed files, each one must be created:
 
-1. Create the `cabinents.yaml` file.
+      - [Create `application_node_config.yaml`](./create_application_node_config_yaml.md)
+      - [Create `cabinets.yaml`](./create_cabinets_yaml.md)
+      - [Create `hmn_connections.json`](./create_hmn_connections_json.md)
+      - [Create `ncn_metadata.csv`](./create_ncn_metadata_csv.md)
+      - [Create `switch_metadata.csv`](./create_switch_metadata_csv.md)
 
-   > If using this file, then do not forget to set the `cabinets-yaml` field in the
-   > [Customize `system_config.yaml`](#33-customize-system_configyaml) step.
+   - For re-installations of CSM 1.3, the previous seed files may be used and this step can be skipped.
+   - For new installations of CSM 1.3 that have prior seed files from CSM 1.2 or older, the following files
+   need to be recreated due to changes in their content or formatting:
 
-   See [Create `cabinets.yaml`](create_cabinets_yaml.md).
+      - [Create `cabinets.yaml`](./create_cabinets_yaml.md)
+      - [Create `hmn_connections.json`](./create_hmn_connections_json.md)
 
-1. Fill in the `ncn_metadata.csv` placeholder values with the actual values.
+   The other files can be re-used (e.g. `application_node_config.yaml`, `ncn_metadata.csv`, and `switch_metadata.csv`).
```

Comment:
This line I think need to be intended one level, so it appears underneath the bullet point correctly
![Screen Shot 2022-07-22 at 11 15 49 AM](https://user-images.githubusercontent.com/77114568/180481163-5b494941-b9ad-4541-a873-9371318c9552.png)
.

Reply:
Yup